Core/CMP API Specification.md

```diff
@@ -234,32 +228,25 @@ The `addEventListener` command can be used to define a callback function (or a p
   <tr>
     <td><code>callback</code></td>
     <td>function</td>
-    <td>function (data: EventListener)</td>
+    <td>function (data: <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/main/Core/CMP%20API%20Specification.md#eventlistener-">EventListener</a>, success: boolean)</td>
```

Comment:
Not sure I fully understand your comment here...
The callback returns the EventListener object as defined just below this section.  I'm not sure what you mean by the callback fro the event vs the callback for the eventlistener object.  

I've added success to the callbacks to align with both the TCF spec and the sample stub code in this GPP spec.  For this command, success would be false if the CMP failed to register the listener for any reason.

Reply:
After re-reading your comment, I think we're on the same page.  The only change here is that instead of directly returning immediately, the callback would be used for that initial response.  The callback would, of course, also continue to be used when future events are triggered.